tests/openwisp2/sample_users/migrations/0003_update_admins_permissions.py

```diff
@@ -0,0 +1,19 @@
+# Manually Created
```

Comment:
I included `update_admins_permissions` in `0002_default_groups_and_permissions.py` on `master`, please remove this (let's try to not add too many migrations in the sample app to keep them easy to scan if needed.